content/pages/docs-as-code.html

```diff
@@ -30,12 +30,13 @@
         <div class="ten wide computer sixteen wide tablet sixteen wide mobile column">
           <div class="ui large left aligned basic vertical segment">
             <h1 class="ui huge header">
-              <em>Docs as code</em> has never been that easy
+              Take the confusion out of <em>docs as code</em>
             </h1>
 
             <p>
-              Treat your documentation in the same way as you treat your code by following the
-              <a href="https://docs-as-co.de/" target="_blank" rel="nofollow">docs as code pattern</a>:
+              Read the Docs is the go-to solution for open source projects and businesses for <em>docs as code</em> hosting.
```

Comment:
```suggestion
              Read the Docs is the go-to solution for open source and businesses projects for <em>docs as code</em> hosting.
```

This sounds easier to read to me, at least.